### PR TITLE
Create zip files for each Ballerina project

### PR DESCRIPTION
**Pull request by pramodya1994**
## Purpose
Create zip files for each Ballerina project. Also removing the code segment adding comments from the source code.
```
// CODE-SEGMENT-BEGIN: segment_1
```

### CODE REVIEW

File: www/pom.xml [hunk 1]
```diff
@@ -144,6 +144,21 @@
...
+            <version>1.7.5</version>
+        </dependency>
+        <dependency>
+            <groupId>org.slf4j</groupId>
+            <artifactId>slf4j-log4j12</artifactId>
+            <version>1.7.5</version>
+        </dependency>
     </dependencies>
 
 </project>
```
Comment (Maninda) [inline www/pom.xml]:
```suggestion
</project>

```

File: docs/learn/integration-tutorials/.gitignore [hunk 2]
```diff
@@ -1 +0,0 @@
-target/
```
Comment (Maninda) [inline docs/learn/integration-tutorials/.gitignore]: Do we need to keep _target_ directory?

File: www/pom.xml (hunk 1 above)
Comment (pramodya1994) [reply www/pom.xml]: Done

File: docs/learn/integration-tutorials/.gitignore (hunk 2 above)
Comment (pramodya1994) [reply docs/learn/integration-tutorials/.gitignore]: This .gitignore file is not needed since `integration-tutorials` is not a Ballerina project.

**Pull request merged**